app/services/subscriptions/create_service.rb

```diff
@@ -20,6 +28,7 @@ def create_from_api(organization:, params:)
       @name = params[:name]&.strip
       @external_id = params[:external_id]&.strip
       @billing_time = params[:billing_time]
+      @subscription_date = params[:subscription_date] || Time.current.to_date
```

Comment:
is it the same as `Time.zone.now` ?

Reply:
yep, from the docs: `Returns Time.zone.now when [Time.zone](https://apidock.com/rails/Time/zone/class) or config.time_zone are set, otherwise just returns Time.now.`